sources/packages/forms/src/form-definitions/studentfinancialinformationappeal.json

```diff
@@ -0,0 +1,386 @@
+                    "tags": [],
+                    "conditional": {
+                        "show": "",
+                        "when": null,
+                        "eq": ""
+                    },
+                    "properties": {},
+                    "lockKey": true
+                },
+                {
+                    "clearOnHide": false,
+                    "key": "panel2Panel2",
+                    "input": false,
+                    "title": "Total unsubsidized day-care costs during class hours for the proposed study period for disabled/infirmed dependents 12 years of age and older Panel",
+                    "theme": "default",
+                    "tableView": false,
+                    "components": [
+                        {
+                            "autofocus": false,
+                            "input": true,
+                            "tableView": true,
+                            "inputType": "text",
+                            "inputMask": "",
+                            "label": "<strong>What are the total unsubsidized day-care costs during class hours for the proposed study period for disabled/infirm dependants 12 years of age and older?</strong>",
+                            "key": "studentNewDaycareCosts12YearsOrOver",
+                            "placeholder": "",
+                            "prefix": "$",
+                            "suffix": "",
+                            "defaultValue": "",
+                            "protected": false,
+                            "persistent": true,
+                            "hidden": false,
+                            "clearOnHide": true,
+                            "delimiter": true,
+                            "decimalLimit": 2,
+                            "requireDecimal": true,
+                            "validate": {
+                                "required": true,
+                                "multiple": "",
+                                "custom": "valid = (input >= 0 && input < data.maxMoneyValue) ? true : 'The number you have entered is outside the expected range.';"
+                            },
+                            "conditional": {
+                                "show": "",
+                                "when": null,
+                                "eq": ""
+                            },
+                            "type": "currency",
+                            "labelPosition": "top",
+                            "tags": [],
+                            "properties": {},
+                            "lockKey": true
+                        }
+                    ],
+                    "type": "panel",
+                    "breadcrumb": "default",
+                    "hideLabel": true,
+                    "tags": [],
+                    "conditional": {
+                        "show": "true",
+                        "when": "studentNewHaveDaycareCosts12YearsOrOver",
```

Comment:
I shared my thoughts about using `new` is the other related PR: https://github.com/bcgov/SIMS/pull/2902#discussion_r1506619290

Reply:
I feel reasonable enough to use the same api keys as application form.